Python/aot_ceval.c

```diff
@@ -1838,30 +1943,52 @@ _PyEval_EvalFrame_AOT_Interpreter(PyFrameObject *f, int throwflag, PyThreadState
 
     co = f->f_code;
 
-    if (can_use_jit && co->co_jit_code == 0 && co->co_opcache_flag >= jit_min_runs /* jit after that many calls or gen yields */) {
-        // JIT assumes opcache is always on
-        if (co->co_opcache_map == NULL) {
-            _PyCode_InitOpcache(co);
-        }
+    if (can_use_jit && co->co_opcache_flag >= jit_min_runs /* jit after that many calls or gen yields */) {
+#ifdef PYSTON_LITE
+        void* code = JIT_FUNC_FAILED;
+        _PyCode_GetExtra((PyObject*)co, code_jitfunc_index, &code);
```

Comment:
I find all this ifdefs and code duplication just to set and retrieve the the pointer to the machine code a bit ugly. What do you think about adding a helper function/macro which handles it?!?